x-pack/plugins/security_solution/server/lib/detection_engine/scripts/rules/queries/query_eql.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "EQL query rule",
+  "description": "Rule with an eql query",
+  "false_positives": [
+    "https://www.example.com/some-article-about-a-false-positive",
+    "some text string about why another condition could be a false positive"
+  ],
+  "rule_id": "rule-id-eql",
+  "enabled": false,
+  "index": [".ds-logs-endpoint.events.process-default-000001"],
+  "interval": "30s",
+  "query": "sequence [process where process.name = \"mimikatz.exe\"] [process where process.name = \"explorer.exe\"]",
+  "output_index": ".siem-signals-default",
```

Comment:
ℹ️  fwiw, we have talked about removing the `output_index` from the REST api and everywhere since the front end doesn't use it and it can be dangerous. This is just informational and historical info. We usually don't use it in the REST examples anywhere other than the kitchen sink/everything ones.

Reply:
We have a ticket open to remove the output_index and cause a break in the API or "soft deprecate it" by having it do nothing other than make logger errors for a release and then remove it. Or we could do an optionally switch where we cause it to throw errors on a release to remove it but if someone really needed it we could give them a switch to add it back.

I'm more of a fan of the "beta" where we just had remove it before GA ;-)